app/locale/en.coffee

```diff
@@ -1539,7 +1539,33 @@
     choose_course: "Choose Course"
     choose_instructions: "Select the course from the dropdown menu you’d like to assign, then click “Assign to Selected Students.”"
     push_projects: "We recommend assigning Web Development 1 or Game Development 1 after students have finished Introduction to Computer Science! See our {{resource_hub}} for more details on those courses."
-  
+    teacher_quest_create_classroom: "Create Classroom"
+    teacher_quest_add_students: "Add Students"
+    teacher_quest_teach_methods: "Help your students learn how to `call methods`."
+    teacher_quest_teach_methods_step1: "Get 75% of at least one class through __Dungeons of Kithgard__"
+    teacher_quest_teach_methods_step2: "[Print out the Student Quick Start Guide in the Resource Hub.](http://files.codecombat.com/docs/resources/StudentQuickStartGuide.pdf)"
+    teacher_quest_teach_strings: "Don't string your students along, teach them `strings`."
+    teacher_quest_teach_strings_step1: "Get 75% of at least one class through __True Names__"
+    teacher_quest_teach_strings_step2: "Use the Teacher Level Selector on Course Guides page to preview __True Names__."
+    teacher_quest_teach_loops: "Keep your students in the loop about `loops`."
+    teacher_quest_teach_loops_step1: "Get 75% of at least one class through __Fire Dancing__."
+    teacher_quest_teach_loops_step2: "Use the __Loops Activity__ in the [CS1 Curriculum guide](https://direct.codecombat.com/teachers/resources/cs1) to reinforce this concept."
+    teacher_quest_teach_variables: "Vary it up with `variables`."
+    teacher_quest_teach_variables_step1: "Get 75% of at least one class through __Known Enemy__."
+    teacher_quest_teach_variables_step2: "Encourage collaboration by using the [Pair Programming Activity](/teachers/resources/pair-programming)."
+    teacher_quest_kithgard_gates_100: "Escape the Kithgard Gates with your class."
+    teacher_quest_kithgard_gates_100_step1: "Get 75% of at least one class through __Kithgard Gates__."
+    teacher_quest_kithgard_gates_100_step2: "Guide students to think through hard problems using the [Engineering Cycle Worksheet](http://files.codecombat.com/docs/resources/EngineeringCycleWorksheet.pdf)."
+    teacher_quest_wakka_maul_100: "Prepare to duel in Wakka Maul."
+    teacher_quest_wakka_maul_100_step1: "Get 75% of at least one class to __Wakka Maul__."
```

Comment:
Many of these could be generalized to "Get __percent__ of at least one class through __levelName__"